sdk/messaging/azservicebus/sample.env

```diff
@@ -1,5 +1,6 @@
 # These values should be filled out when testing.
 SERVICEBUS_CONNECTION_STRING=<service bus connection string>
+SERVICEBUS_CONNECTION_STRING_NO_MANAGE=<service bus connection string with listen and send permissions only>
```

Comment:
This file is a good idea, thanks for maintaining it

Reply:
Carried this over from the JS team. I think technically there should be an .env file for samples (ie, users) and one for developers of the package itself. We can figure that out over time.